docs/howtoguides/configure_proxies.rst

```diff
@@ -10,6 +10,12 @@ addition, the Ubuntu Pro Client will automatically set up proxies for all
 programs required for enabling Ubuntu Pro services. This includes APT, snaps,
 and Livepatch.
 
+.. note::
+
+    Throught this guide, we will set the https proxy with an http proxy.
+    However, depending on your proxy setup, you might need to configure the
+    Pro https proxy with a https prefix.
```

Comment:
@s-makin I haven't applied that change everywhere, since I believe the prefix part should still be lowercase. But let me know if you disagree

Reply:
If you want to leave it in lowercase, you'll need to either add it to the spelling exception list or decorate it with backticks, because it's currently preventing the spelling check from passing. Whichever way you go, feel free to merge after the checks pass :)